### PR TITLE
Course view

### PR DESCRIPTION
**Pull request by Kingofikings441**
Course-constraints is finish if there is a problem let me know

### CODE REVIEW

File: app/services/csv_handler.rb [hunk 1]
```diff
@@ -8,82 +8,129 @@ def upload(file)
...
-      return { alert: "There was an error uploading the CSV file: #{e.message}" }
     end
+    # Flash data to be received by controller
+    { notice: 'Rooms successfully uploaded.' }
+  rescue StandardError => e
+    { alert: "There was an error uploading the CSV file: #{e.message}" }
   end
 
   def parse_instructor_csv(schedule_id)
-    begin
```
Comment (wahibkapdi) [inline app/services/csv_handler.rb:37]: Why was begin removed?

File: app/helpers/courses_helper.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module CoursesHelper
```
Comment (wahibkapdi) [inline app/helpers/courses_helper.rb:3]: Delete if not required.

File: app/models/course.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Course < ApplicationRecord
+  belongs_to :schedule
```
Comment (wahibkapdi) [inline app/models/course.rb:4]: Shouldn't this also have a has_many for sections?

**Review by wahibkapdi — CHANGES_REQUESTED** [review]
Make sure the courses table also shows sections, and their individual seat expectations

File: app/services/csv_handler.rb (hunk 1 above)
Comment (Kingofikings441) [reply app/services/csv_handler.rb:37]: I ran rubocop -a that might be why, I never took that line on purpose

File: app/models/course.rb (hunk 3 above)
Comment (Kingofikings441) [reply app/models/course.rb:4]: your correct I have added it

**Review by wahibkapdi — APPROVED** [review]

**Pull request merged**